cargo-dist/templates/ci/github/release.yml.j2

```diff
@@ -133,6 +133,18 @@ jobs:
         with:
           name: cargo-dist-cache
           path: ~/.cargo/bin/dist
+      {{%- if need_cargo_auditable %}}
+      - name: Install cargo-auditable
+        # we specify bash to get pipefail; it guards against the `curl` command
+        # failing. otherwise `sh` won't catch that `curl` return non-0
+        run: {{{ install_cargo_auditable_sh }}}
+        shell: bash
+      - name: Cache cargo-auditable
```

Comment:
Feel free to skip this cache. We use this for cargo-dist itself because we reuse the same binary in multiple different jobs running on the same architecture, but the same isn't true for cargo-auditable.